### PR TITLE
[usecase] Add test for xwalk_view member in manifest file

### PR DESCRIPTION
**Pull request by wanghongjuan**
Impacted TCs num(approved): New 1, Update 0, Delete 0
Unit test Platform: Android
Android test result summary: Pass 1, Fail 0, Block 0

https://crosswalk-project.org/jira/browse/XWALK-4963

### CODE REVIEW

File: usecase/usecase-wrt-android-tests/steps/ManifestDemo7/step.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+var step = '<p>Purpose:</p>'
+            +'<p>Verify the manifest members xwalk_view can work normally when launch the test app</p>'
```
Comment (Honry) [inline usecase/usecase-wrt-android-tests/steps/ManifestDemo7/step.js]: 's/members/member'

**Comment by Honry** [comment]
@wanghongjuan, can this test be automate by BDD?

**Comment by wanghongjuan** [comment]
Updated

**Comment by Honry** [comment]
LGTM

**Pull request merged**